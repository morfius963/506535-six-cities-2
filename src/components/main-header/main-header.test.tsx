import React from "react";
import renderer from "react-test-renderer";
import MainHeader from "./main-header.jsx";

jest.mock(`react-router-dom`, () => ({
  Link: () => null
}));

describe(`snapshot test`, () => {
  it(`Component correctly renders`, () => {
    const props = {
      userData: {
        name: `Vitalii`,
        avatar: ``
      },
      isAuthorizationRequired: true
    };

    const tree = renderer
      .create(<MainHeader {...props} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
