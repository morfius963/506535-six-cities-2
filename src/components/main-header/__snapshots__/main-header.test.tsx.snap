// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test Component correctly renders 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__wrapper"
    >
      <div
        className="header__left"
      />
      <nav
        className="header__nav"
      >
        <ul
          className="header__nav-list"
        >
          <li
            className="header__nav-item user"
          />
        </ul>
      </nav>
    </div>
  </div>
</header>
`;
